ui/model/__init__.py

```diff
@@ -35,7 +33,7 @@ class Model:
     timeout_move: Move = None
     history: GameHistory = field(default_factory=GameHistory)
     game: Game = field(default_factory=Game)
-    config: Config = field(default_factory=Config.from_default)
+    config: config.Config.Config = field(default_factory=config.Config.from_default)
```

Comment:
I feel like this can be changed to a `from` import lol. I'm not sure what the nested classes are about

Reply:
I think I was having circular reference problems when using a from import. Not sure if there is a better way to fix this.